devices.js

```diff
@@ -11357,6 +11357,21 @@ const devices = [
         exposes: [e.occupancy(), e.battery_low(), e.tamper(), e.battery(), e.temperature(), e.illuminance(), e.illuminance_lux(),
             e.humidity()],
     },
+    {
+        zigbeeModel: ['ColorTemperature'],
+        fingerprint: [{modelID: '07073L', manufacturerName: 'Seastar Intelligence'}],
+        model: '07073L',
+        vendor: 'Immax',
+        description: 'Neo CANTO/HIPODROMO SMART, color temp, dimmable, Zigbee 3.0',
+        extend: preset.light_onoff_brightness_colortemp(),
```

Comment:
Can you check the database.db entries for these devices to add the colorTempRange info?
Some tips on how to grab the info can be found here https://github.com/Koenkk/zigbee-herdsman-converters/pull/2357#issuecomment-797574094 you might need to tweak the ep though.